### ISSUE TITLE
Accessibility: respect prefers-reduced-motion

### ISSUE THREAD

**Issue by joedolson** [Automattic/tour#29]
The tour trigger has a continuously running animation that doesn't respect a user's "prefers-reduced-motion" settings. The pulsing glow should be static if a user prefers reduced animations in their OS.
